data/watch.yml

```diff
@@ -1,151 +1,438 @@
+- name: 'Parafuzz: Coverage-guided Property Fuzzing for Multicore OCaml programs'
+  description: "We develop ParaFuzz, an input and concurrency fuzzing tool for Multicore
+    OCaml programs. ParaFuzz builds on top of Crowbar which combines AFL-based grey
+    box fuzzing with QuickCheck and extends it to handle parallelism.\r\n\r\nSumit
+    Padhiyar\r\nIndian In..."
+  embedPath: /videos/embed/c0d591e0-91c9-4eaa-a4d7-c4f514de0a57
+  thumbnailPath: /static/thumbnails/b1a67c0a-2eb9-4c9d-97ce-b4dc039da59b.jpg
+  date: 2021-08-27T10:36:13.000Z
+  language: English
+  category: Science & Technology
+- name: Deductive Verification of Realistic OCaml Code
+  description: We present the formal verification of a subset of the Set module from
+    the OCaml standard library. The proof is conducted using Cameleer, a new tool
+    for the deductive verification of OCaml code. Cameleer takes as input an OCaml
+    program, annotated u...
+  embedPath: /videos/embed/92309d92-8cbf-4545-980c-209c96e42a79
+  thumbnailPath: /static/thumbnails/db29d3bf-784c-47f1-bf90-f13fd2a9d530.jpg
+  date: 2021-08-27T10:33:38.000Z
+  language: English
+  category: Science & Technology
+- name: Adapting the OCaml ecosystem for Multicore OCaml
+  description: "OCaml 5.0 with support for shared-memory parallelism being around
+    the corner, there\u2019s increasing interest in the community to port existing
+    libraries to Multicore. This talk will take the attendees through what the arrival
+    of Multicore means to th..."
+  embedPath: /videos/embed/629b89a8-bbd5-490d-98b0-d0c740912b02
+  thumbnailPath: /static/thumbnails/41212c84-36d6-44fa-b884-2ced79505929.jpg
+  date: 2021-08-27T10:18:38.000Z
+  language: English
+  category: Science & Technology
+- name: 'OCaml and Python: Getting the Best of Both Worlds'
+  description: In this talk we present how we expose a wide variety of OCaml libraries
+    and services so that they can be accessed from Python. Our initial use case on
+    the Python side consisted in Jupyter notebooks used to analyse various datasets,
+    these datasets ...
+  embedPath: /videos/embed/9eafdb1e-9be9-4a52-98b4-f4696eda4c18
+  thumbnailPath: /static/thumbnails/53db83d1-0763-4dfa-a087-cce3cad1b581.jpg
+  date: 2021-08-27T10:16:54.000Z
+  language: English
+  category: Science & Technology
+- name: 'GopCaml: A Structural Editor for OCaml'
+  description: "This talk presents GopCaml-mode, the first structural editing plugin
+    for OCaml. We will give a tour of the main plugin features, discussing the plugin\u2019s
+    internal design and its integration with existing OCaml and GNU Emacs toolchains.\r\n\r\nKiran
+    Gop..."
+  embedPath: /videos/embed/e0a6e6f2-0d40-4dfc-9308-001c8e0f64d6
+  thumbnailPath: /static/thumbnails/729cda51-abb3-4c8d-9d0e-b922e81be518.jpg
+  date: 2021-08-27T10:12:58.000Z
+  language: English
+  category: Science & Technology
+- name: State of the OCaml Platform 2020
+  description: 'This talk covers:
+
```

Comment:
Are the multiple linebreaks expected here? They weren't there in the previous data from what I can see.

Reply:
Not sure, I reverted back to old versions of things (in particular `yaml.2.1.0`) and it still added the line-breaks. Now we have the `updated_at` timestamp, looks like someone might have gone in to edit that description of the video perhaps: https://github.com/ocaml/ood/blob/d104adfcc99240649000a44d1c6c695137257b2a/data/watch.yml#L404

I'll keep my eye on it in the meantime but I don't think it's on us, with the `updated_at` it will be easier to track going forward :))